Entity/AbstractEntity.php

```diff
@@ -28,6 +28,16 @@ public function getCreated()
     }
 
     /**
+     * Set updated date
+     *
+     * @param \Datetime $date The updated date
+     */
+    public function setUpdated($date)
```

Comment:
ok I see, you mean if you only change static files for instance, the bdd is not modified and gedmo is not called at all. What a pity!